network/components/virt_tx.c

```diff
@@ -62,9 +62,10 @@ void tx_provide(void)
                     assert(!err);
                     continue;
                 }
-
+                #ifdef CONFIG_ARCH_ARM
                 cache_clean(buffer.io_or_offset + state.buffer_region_vaddrs[client],
                             buffer.io_or_offset + state.buffer_region_vaddrs[client] + buffer.len);
+                #endif
```

Comment:
Why is this not needed?

Reply:
Same reason as `cache_clean_and_invalidate` is not needed.